### ISSUE TITLE
Add Stock/Warehouses to isotope

### ISSUE THREAD

**Issue by ghost** [isotope/core#341]
I know, there's something in SVN but it's one of the hardly missed features.
:-D

_--- Originally created by psren on 2012-01-13 23:13:52 (ID 2800)_

**Comment by CodeAlDente** [comment]
:+1: +1

**Comment by CodeAlDente** [comment]
It seems like a lot of people are asking for this feature and I think it would be a huge improvement. Why don't you start a crowdfunding campaign for that?

**Comment by aschempp** [comment]
> Why don't you start a crowdfunding campaign for that?

We already did a few years ago, but unfortunately it was not successful…

**Comment by CodeAlDente** [comment]
> We already did a few years ago, but unfortunately it was not successful…
That's too bad. Is there any way to realize a simple solution? In my case it would be enough to have a form field with the available amount of the articles. When a customer buys an article, the number should be substracted by the number of articles he bought. And if the article is out of stock it should disappear from the shop. Maybe also with an error message on the checkout if the customer buys more from the article than I have in stock. Is this possible?

**Comment by aschempp** [comment]
That sure is possible, but you need to find someone that develops it for you ;-)

**Comment by CodeAlDente** [comment]
Hm, I see. But wouldn't it be enough to create an attribute for this and to use the hooks to update the value of the field? And after a bit of (dirty) magic it should work. Not that nice and also not update safe, but it would work I think.

**Comment by aschempp** [comment]
Maybe. If that's enough for your use case. But the version we'll implement in core will certainly need more features.

**Comment by CodeAlDente** [comment]
Yep, that sounds much better. How much money do you guys need to realize it?

**Comment by aschempp** [comment]
Please use our contact options for project requests: https://isotopeecommerce.org/en/contact.html
